src/back/util/db.js

```diff
@@ -7,15 +7,43 @@ const Database = {
   users: generateUserBase(),
 };
 
-export const addUser = (user) => {
-  Database.users = [
-    ...Database.users,
-    user,
+/**
+ * Clears all of the database 'tables'.
+ */
+export const clear = () => {
+  const keys = Object.keys(Database);
+  keys.forEach((key) => {
+    Database[key] = [];
+  });
+};
+
+/**
+ * Adds an element to the Database object.
+ * @param {String} key The database name in which the element will be added
+ * @param {*} elem Element to be added.
+ */
+export const add = (key, elem) => {
+  if (!key || !elem || !Database[key]) {
+    return; // nothing to do. Won't add anything.
+  }
+
+  Database[key] = [
+    ...Database[key],
+    elem,
   ];
 };
 
-export const removeUser = (userId) => {
-  Database.users = Database.users.filter(user => user.id !== userId);
+/**
+ * Removes an element by ID
+ * @param {string} key The database name in which the element will be removed.
+ * @param {number} id ID of the element to be removed
+ */
+export const removeById = (key, id) => {
```

Comment:
if **key** relates to the database table, why name it **key**?? name it **database** or **table** or **schema**. Same applies above. on the **add(key, elem)** function